keyvault/data-plane/azure-keyvault-cryptography/src/main/java/com/microsoft/azure/keyvault/cryptography/AsymmetricEncryptionAlgorithm.java

```diff
@@ -73,5 +73,5 @@ protected AsymmetricEncryptionAlgorithm(String name) {
      * @throws NoSuchAlgorithmException
      * @throws NoSuchPaddingException
      */
-    public abstract ICryptoTransform CreateDecryptor(KeyPair keyPair, Provider provider) throws InvalidKeyException, NoSuchAlgorithmException, NoSuchPaddingException;
+    public abstract ICryptoTransform createDecryptor(KeyPair keyPair, Provider provider) throws InvalidKeyException, NoSuchAlgorithmException, NoSuchPaddingException;
```

Comment:
We almost certainly shouldn't rename these methods - unfortunately they are public API.

Reply:
Would I want to add these as suppressions instead?

Reply:
Sadly, yes. Whenever you add a suppression please leave a message stating why the suppression exists - so say something like "public API already released with incorrect upper-case method naming pattern"